### PR TITLE
Update mac install steps

### PR DESCRIPTION
**Pull request by ellennickles**
Adding info to help users find Macintosh HD so they can complete steps for Mac installation of this plugin

### CODE REVIEW

**Review by matamalaortiz — COMMENTED** [review]
Oh, I think the path was wrong. 

It should be 
`~/Library/Application\ Support/Adobe/CEP/extensions` 
rather than 
` ~/Library/Application Support/Adobe/CEP/extensions`
(missing a back slash \ )

Maybe that's why they couldn't find it.

Then maybe we can say :

**Mac** 
Open the terminal and paste: `cd ~/Library/Application\ Support/Adobe/CEP/extensions` 

What do you think?

**Comment by ellennickles** [comment]
The original path is right! The people I've helped install this plugin do not know what the terminal is or how to open it. They are very familiar with Photoshop, but since they do not know about terminal and since Macintosh HD does not appear on the Desktop by default any longer, they get stuck at this installation step and don't know what to do.

**Comment by matamalaortiz** [comment]
gotcha! this is for users who are trying to locate the folder throw the GUI. Lgtm!

**Comment by matamalaortiz** [comment]
cd'ing into the current path doesn't work for me though, does it work for you when you copy paste it? If not, will change it to 
`cd ~/Library/Application\ Support/Adobe/CEP/extensions`
rather than
`cd ~/Library/Application Support/Adobe/CEP/extensions`
for those wanting to find it via the terminal.

**Comment by ellennickles** [comment]
> gotcha! this is for users who are trying to locate the folder throw the GUI. Lgtm!

Ok great!

**Comment by ellennickles** [comment]
> cd'ing into the current path doesn't work for me though, does it work for you when you copy paste it? If not, will change it to
> `cd ~/Library/Application\ Support/Adobe/CEP/extensions`
> rather than
> `cd ~/Library/Application Support/Adobe/CEP/extensions`
> for those wanting to find it via the terminal.

`cd /Library/Application\ Support/Adobe/CEP/extensions' works for me if I'm already in my root directory, so I agree that we should add a `\` to take care of that space in the Application Support folder

File: README.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ Unzip the plugin and copy the entire `RunwayML` directory to Photoshop Extension
 
 **Mac:** ~/Library/Application Support/Adobe/CEP/extensions
```
Comment (ellennickles) [inline README.md]:
```suggestion
**Mac:** ~/Library/Application\ Support/Adobe/CEP/extensions
```

**Review by ellennickles — COMMENTED** [review]
Updating with change suggested by @matamalaortiz

**Comment by ellennickles** [comment]
@matamalaortiz Except that I just realized that adding a `\` might be really confusing for users who are looking for the directory through the GUI. So maybe we should leave that out because I expect that developers will know how to handle that? What do you think?

**Pull request merged**